collaborative/IOU/heat_index.ipynb

```diff
@@ -162,218 +165,426 @@
    "metadata": {
     "tags": []
    },
    "outputs": [],
    "source": [
-    "heatidx_hist_day = heatidx_hist_hour.resample(time='1D').max() # daily max\n",
-    "# heatidx_hist_day = heatidx_hist_hour.resample(time='1D').median() # daily median"
+    "# counts the number of hours in each day above the heat index threshold\n",
+    "num_heatidx_histhours = (heatidx_hist_hour >= hi_threshold).resample(time='1D').sum()\n",
+    "num_heatidx_histhours.name = 'Hours per day above Heat Index threshold of {}°F'.format(hi_threshold)"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "dea35a78-b63c-4b61-ad0f-ee709fd76072",
+   "id": "52685577-e904-4216-a320-82468d680e45",
    "metadata": {},
    "source": [
-    "Let's visualize the historical trend: "
+    "Let's identify one year to visualize the cycle in Heat Index temperatures. We are selecting 2000 here, but the commented out line of code below illustrates how to look at all of the data here. Just comment out the first line under visualize by adding a `#` symbol, and uncommenting the line below by removing the `#` symbol."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "f99ee702-9dd0-404f-8b0c-b59adfac5c06",
-   "metadata": {},
+   "id": "29d35f96-b811-4f7a-938f-1c54b40f1c15",
+   "metadata": {
+    "tags": []
+   },
    "outputs": [],
    "source": [
-    "ck.view(heatidx_hist_day)"
+    "data_one_year = num_heatidx_histhours.sel(time=\"2000\")\n",
+    "\n",
+    "# visualize\n",
+    "data_one_year.hvplot.line(x='time', by='simulation') # a specific year\n",
+    "# num_heatidx_histhours.hvplot.line(x='time', by='simulation') # all years"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "30df5679-25ff-42f0-8128-54e10ac33a37",
+   "id": "5b38dd97-c64d-4f8b-8017-c606aefb80e4",
    "metadata": {},
    "source": [
-    "In the plot above, we visualize the daily max heat index values through the historical summer period. You'll note that in Fresno, there are daily values below 80°F. [Heat Index](https://www.weather.gov/ama/heatindex) typically only becomes \"noticeable\" to people above 80°F in terms of heat safety caution. We'll return to this in Step 2. \n",
+    "Heat Index values above our selected threshold (default is 80°F) begin to pick up in March for Fresno, and tail off in November. While high Heat Index values are critical to know during the summer months, we should be aware of high Heat Index values throughout the entire year as well. \n",
     "\n",
-    "#### 1c) Retrieve and calculate the projected trends\n",
-    "Next we will repeat the same data retrieval and daily max process as we did for the historical data so we can compare. Like the historical data, retrieving the hourly projections data will also take a few minutes - hang tight!"
+    "### 2b) Sum the total hours per year\n",
+    "It may also be useful to know how many hours in the entire year above the threshold to see the trends over time. We'll calculate this next."
```

Comment:
```suggestion
    "It may also be useful to know how many hours in the entire year are above the threshold in order to see the trends over time. We'll calculate this next."
```